core/shared/src/main/scala-2.11/zio/clock.scala

```diff
@@ -16,8 +16,9 @@
 
 package zio
 
-import zio.duration.Duration
```

Comment:
Really minor thing can you order imports? Then let's get this merged!

Reply:
done!